content/docs/introduction/design-doc.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Design Documents
+sort_rank: 7
+---
+
+# Design documents
+
+When contributing large changes or features to the Prometheus ecosystem, design
+documents are written by individuals or groups. Those documents are proposed for
+review and approval to the community.
+
+This page is a list of the design documents we know of. If you create a new
+design document, or some documents are not listed, please open a [pull
+request](https://github.com/prometheus/docs/) to add it to this list.
+
+Design doc dot not always reflect exactly what has been implemented, and
+implementation details might have changed since a feature was merged. Design
+docs are not considered documentation.
+
+If you want to pick a design doc in `TODO` or implement a proposed design doc,
+please reach out to the [developers mailing list](/community) first to make sure
+no one else has picked that task already, and the design doc is approved and
+still relevant.
+
+<table>
+  <tr><th>Document</th><th>Initial date</th><th>Status</th><th>Pull requests</th></tr>
+  <tr><td><a href="https://github.com/prometheus/alertmanager/blob/master/doc/design/secure-cluster-traffic.md">Secure Alertmanager cluster traffic</a></td><td>2019-02-21</td><td>Approved</td><td><a href="https://github.com/prometheus/alertmanager/pull/2237">alertmanager#2237</a></td></tr>
+  <tr><td>TSDB Head Improvements (<a href="document 1">https://docs.google.com/document/d/184urkLQnM7rqLmGvS66I15jU2pyPk_qwd8v_qDlXczo/edit#heading=h.ow34omg0frla</a> - <a href="document 2">https://docs.google.com/document/d/1pnEsxB0CDLOxQipGw_vhkJpoDZfZPs_KjqCriumDAXQ/edit#heading=h.qsf5sgqoqemc</a>)</td><td>2019-12-09</td><td>Partially implemented</td><td></td></tr>
```

Comment:
Link text and URL are switched here. (URL needs to be the `href` and "document 1/2"  needs to be the text content of the tag.)